cmd/aries-js-worker/package-lock.json

```diff
@@ -446,6 +446,14 @@
       "integrity": "sha512-Wm6ukoaOGJi/73p/cl2GvLjTI5JM1k/O14isD73YML8StrH/7/lRFgmg8nICZgD3bZZvjwCGxtMOD3wWNAu8cg==",
       "dev": true
     },
+    "axios": {
```

Comment:
why was this file change as part of this PR?

Reply:
not sure, but it resulted from `make all`